.circleci/config.yml

```diff
@@ -100,7 +100,7 @@ jobs:
           template: basic_fail_1
 
 workflows:
-  version: 2.1
+  version: 2
```

Comment:
Any reason for dropping back to version 2 from 2.1? I think 2.1 is needed to run local builds (which still work fine with docker under linux).

Reply:
I was informed that 2.1 was an invalid value under the schema @bhamail - only reason I changed it!

Reply:
See https://circleci.com/docs/configuration-reference/#workflows

Reply:
Odd, and confusing: https://circleci.com/docs/configuration-reference/#version

Later, I'll try to build locally under linux with the 2 value.

Reply:
Note that is the workflow version, not the file version - which is `2.1`